### ISSUE TITLE
For the nmos-cpp-registry to with with unicast DNS-SD 

### ISSUE THREAD

**Issue by andywangevertz** [sony/nmos-cpp#362]
I have a naive quesiton and hopefully you could share some lights on this.

I understand that the nmos-cpp-registry could use the 'local' domain and using the mDNS and all works. The easy-nmos is an perfect example for it.

Now I try to setup the nmos-cpp-registry with the external DNS server (BIND9 on ubuntu) to use the unicast DNS-SD.  How I can configure it to use the external DNS server. I checked the registry.json and I can not find any parameter related to configure the DNS. So If I add  "domain":"testsuite.nmos.tv" in the registry.json and it will know that it is using the unicast DNS-SD and will use the system DNS resolver?

I tried about setup with the easy-nmos and now on the http://registry/admin page, it could only see itself so I guess that for the virtual-node, I have to config it to find the external DNS server in order to register properly? Am I on the right track?

Thanks!

**Comment by garethsb** [comment]
> So If I add "domain":"testsuite.nmos.tv" in the registry.json and it will know that it is using the unicast DNS-SD and will use the system DNS resolver?

Yes (precisely it can depend on the configuration of mDNSResponder or Avahi, but basically, yes)
